### ISSUE TITLE
Error: Dimension mismatch error when using the example code to align <FACET 4.1> to <glove_vectors>.

### ISSUE THREAD

**Issue by hyang428** [A2Zadeh/CMU-MultimodalSDK#178]
Hi,

First, appreciate the effort to release this SDK, which makes the data preparation much easier.
It will be much better if there is any document related to the features, video sturctures, functions, and so on.

I was able to download the CMU-MOSI dataset.
However, an error happened when aligning <FACET 4.1> to <glove_vectors> using the example code. 
Mode details
`RuntimeError: <Error>: Dimension mismatch between intervals and features when aligning <FACET 4.1> computational sequences to <glove_vectors> computational sequence`

As this is a high-level code, it is difficult to locate and fix the problem.
Also, it seems the tutorial mentioned under the Readme use a different way to load and process the dataset, which is somehow different with the method used in the example code.
It confused me which one is the good way to load and align the dataset,

Thanks.

**Comment by A2Zadeh** [comment]
@hyang428 Is the issue resolved? Let me know and I can reopen it.

**Comment by hyang428** [comment]
Thanks @A2Zadeh. I just made a mistake in the code, and I have fixed the problem.

By the way, the [tutorial](https://github.com/Justin1904/CMU-MultimodalSDK-Tutorials) and the example code seems use different method to process the dataset, and even a different "CMU_MOSI_ModifiedTimestampedWords". Which one should I follow, should I assume the SDK always ship with the up-to-date features and function? thanks again
